app/models/miq_cockpit_ws_worker/runner.rb

```diff
@@ -0,0 +1,65 @@
+class MiqCockpitWsWorker::Runner < MiqWorker::Runner
+  BINDING_ADDRESS = ENV['BINDING_ADDRESS'] || (Rails.env.production? ? nil : "0.0.0.0")
+
+  def self.start_worker(*args)
+    runner = MiqCockpitWsWorker::Runner.new(*args)
+
+    runner.prepare
+    runner.start_cockpit_ws
+    runner.run
+  end
+
+  def do_work
+  end
```

Comment:
The empty method is not necessary (unless you are trying to *avoid* what the parent does, and if so, put a comment in there saying so)